handler/src/test/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngineTest.java

```diff
@@ -54,4 +56,14 @@ protected void cleanupServerSslContext(SslContext ctx) {
     protected void cleanupServerSslEngine(SSLEngine engine) {
         ReferenceCountUtil.release(engine);
     }
+
+    @Test(expected = NullPointerException.class)
+    public void testNotLeakOnException() throws Exception {
```

Comment:
this produced a leak and so would fail the build when run with `-Pleak` on the ci.